### DISCUSSION TITLE
Debugging .jira/config.yml

### DISCUSSION THREAD

**Discussion by jrosales** [atlassian/github-for-jira#1571] [General] [answered]
After validating:
- Connection is correct
- Deployment events are being received and displayed in Jira
- Config file is valid yaml

We are still unable to map our environments to Jira's environments.

Is there a way to debug if:
 - config.yml was found by the app
 - Jira is able to parse the content
 - Jira was able to match and map an environment

**Comment by jrosales** [comment, marked as answer]
It looks like the env mapping is not rolled out yet. https://github.com/atlassian/github-for-jira/issues/1303#issuecomment-1226715001
@thombergs I am interested in enabling this for our cloud host. Please message me so I can provide details.

**Reply by thombergs** [reply]
Hey @jrosales yes, sorry, I just activated the feature for 20% of the users last Friday. I just need your jira host name (`example.atlassian.net`) to activate it for you. If you don't want to share that here, you can send an email to thombergs@atlassian.com.

**Comment by aldenquimby** [comment]
@thombergs is this feature now globally enabled? I am having the same problem. Here is the `.jira/config.yml` from my default branch:
```yml
deployments:
  environmentMapping:
    development: ["local"]
    staging: ["qa", "uat"]
    production: ["production"]
    testing: ["*"]
```

But it doesn't seem to be working. `qa` and `uat` still show as `testing`, and all other environments (e.g. `alden` or `pr27`) show Unknown instead of `testing`.

Is there any way for me to debug and see what's going wrong with the integration?
